### DISCUSSION TITLE
[Question] Physical Material of deformable objects

### DISCUSSION THREAD

**Discussion by Entongsu** [isaac-sim/IsaacLab#758] [Q&A]
Hi,

I am wondering there is any methods to obtain and set the physical parameter of the physical material, such as youngs_moduli, elasticity_dampings during the simulation. Thank you!

**Comment by Mayankm96** [comment]
Unfortunately, there are none that I am aware of. I usually just look up the material properties online and use those values in simulation. It worked fine for the experiment I did back for the paper. However, it is hard to comment on the general "sim-to-real" gap for deformables right now.

If there is a benchmark to compare the fidelity in general, we are happy to take a look and pass it on to the PhysX team.

**Reply by Entongsu** [reply]
Hi,

I have tried to spawn_deformable_body_material to reset the physical parameter in every reset. However, I have found that the material deformable is on the CPU instead of the GPU, and no matter how I change these values, the simulation result is nearly the same.
